Cookbook/Technical-Documents/Current-guide.md

```diff
@@ -0,0 +1,29 @@
+# `Current` Guide - how to use it in Babylon iOS Project
+
+### Where is that? 
+- `Current` is the global instance of the `World` type which is located in the `BabylonDependencies.framework`. 
+Its purpose is to provide convenient access to the instances that should be shared between different parts of the app (cross-cutting concerns). For more information please read the [How Control the World](/Cookbook/Proposals/ControlTheWorld.md) document.
+
+## `Do`s and `Don't`s
+
+#### Do
+- access `Current` **only** from topmost `AppDelegate` until `Builder`s layer
+- pass the required dependencies down the chain;
```

Comment:
```suggestion
- Pass the required dependencies down the chain. The builder accesses Current and it injects the specific dependencies down the chain
```

Reply:
It's repeating "down the chain" twice.

Reply:
also as items I prefer to have them as 
- lowercase

Reply:
Items in a list are usually uppercase https://www.quickanddirtytips.com/education/grammar/formatting-vertical-lists

Reply:
List items AFAIK are uppercase. 🤔

Reply:
@diegopetrucci from the link you provided I see a capitalised list 👍 :

<img width="228" alt="Screen Shot 2019-06-11 at 14 50 59" src="https://user-images.githubusercontent.com/17405823/59277576-5c983380-8c58-11e9-9e4e-2b65ddedb3e1.png">

But giving a quick read... I clicked the link Chicago Manual Style... https://www.quickanddirtytips.com/education/grammar/grammar-style-issues

> One book discusses capitalization, another partially discusses punctuation, and so on. Only the [Chicago Manual of Style](https://www.quickanddirtytips.com/education/grammar/grammar-style-issues) gives the topic the respect it deserves. 

... I see a lowercased list 

<img width="354" alt="Screen Shot 2019-06-11 at 14 52 53" src="https://user-images.githubusercontent.com/17405823/59277780-ac76fa80-8c58-11e9-9ddc-6af588ea2838.png">

I change it as long it adds value to the document and improve the understanding.